### ISSUE TITLE
Doc site ssl cert is expired

### ISSUE THREAD

**Issue by dannysauer** [johnolafenwa/DeepStack#180]
Not a traditional code issue, but... 
The doc site at https://docs.deepstack.cc/ has a cert that expired a month ago.  Might wanna fix that, and also see about setting up one of the free monitoring tools. :)

**Comment by diamondbarcode** [comment]
yes i notice it today when i see to see if any update .. first i tough project is death or something ...

**Comment by tbjers** [comment]
@johnolafenwa @OlafenwaMoses y'all still alive? SSL certificates on both your sites are expired, for quite some time now.

**Comment by tjbenator** [comment]
I'm receiving an error when trying to download from the docs page as well:
https://deepquest.sfo2.digitaloceanspaces.com/deepstack/DeepStack-Installer-CPU-2022.01.1.exe
```
<Error>
<Code>NoSuchBucket</Code>
<BucketName>deepquest</BucketName>
<RequestId>tx000001c7ebb47967859b1-0063b641cb-35d1f25-sfo2b</RequestId>
<HostId>35d1f25-sfo2b-sfo2-zg02</HostId>
</Error>
```

EDIT: I guess they are also tagged thankfully on github: https://github.com/johnolafenwa/DeepStack/tags

**Comment by nefh** [comment]
The website seems dead for some time, is DeepStack over?

**Comment by tbjers** [comment]
> The website seems dead for some time, is DeepStack over?

Maybe the repo owner wrote the best AI of all time, got assassinated, then ended up being resurrected into a human-machine hybrid chat bot that had to keep fielding questions about where to buy cocaine... Or, maybe they got poisoned with radioactive material by a Russian spy to prevent them from asking ChatGPT the one burning question: what does Putin wear to bed?

**Comment by VorlonCD** [comment]
> > The website seems dead for some time, is DeepStack over?
> 
> Maybe the repo owner wrote the best AI of all time, got assassinated, then ended up being resurrected into a human-machine hybrid chat bot that had to keep fielding questions about where to buy cocaine... Or, maybe they got poisoned with radioactive material by a Russian spy to prevent them from asking ChatGPT the one burning question: what does Putin wear to bed?

I think an AI based troll wrote this :smirk:

[CodeProject.AI](https://github.com/codeproject/CodeProject.AI-Server) is a drop in replacement for deepstack and is being actively developed.

**Comment by tbjers** [comment]
> > > The website seems dead for some time, is DeepStack over?
> > 
> > 
> > Maybe the repo owner wrote the best AI of all time, got assassinated, then ended up being resurrected into a human-machine hybrid chat bot that had to keep fielding questions about where to buy cocaine... Or, maybe they got poisoned with radioactive material by a Russian spy to prevent them from asking ChatGPT the one burning question: what does Putin wear to bed?
> 
> I think an AI based troll wrote this 😏

If I was an AI you would be in real trouble. Furthermore, I have a hunch that you think that "troll" means "those that disagree with my opinion." If I was a troll, would I be joking about what might have happened to the repo maintainers? No. I would be making inflammatory statements that would be contrary to those of others in the given context, such as "I love Amiga!" on an Atari bulletin board.

**Comment by johnolafenwa** [comment]
https://github.com/johnolafenwa/DeepStack/issues/182
